src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.Managed.DesignTime.targets

```diff
@@ -52,6 +52,11 @@
     <DebuggerFlavor Condition = "'$(DebuggerFlavor)' == ''">ProjectDebugger</DebuggerFlavor>
   </PropertyGroup>
 
+  <!-- For rule files that don't need localization we use the neutral directory to save duplicating files into each language -->
+  <PropertyGroup Condition="'$(ManagedXamlNeutralResourcesDirectory)' == ''">
```

Comment:
I'm thinking that English above should fall back to `ManagedXamlNeutralResourcesDirectory`, so that to control this directory you only need to set one property.

Reply:
Yeah, that makes sense. It might even make sense, in light of this change, to create an `\en\` subdirectory for the English files, just so that neutral can really mean "has no translations", rather than "has no translations or is English".

Reply:
In VS, neutral almost always means English. In Windows, they do put English in its own directory.

Reply:
In that case lets leave it for another day, since this is a P2 bug